### DISCUSSION TITLE
I no longer have permission to upload images to Firebase Storage

### DISCUSSION THREAD

**Discussion by Ken-JIAnomads** [AntlerVC/firetable#1235] [Support / Q&A] [answered]
I was able to upload images via Rowy to my Firebase Storage but no longer.  

I have my Flutterflow project connected to Rowy. I think I must have overwritten exisfirebase storage rules when i deployed it from flutterflow so users can upload their images in the app.  

Can I add the original Rowy rules back in ? 

Thanks in advance!

![Screenshot 2023-05-02 at 11 32 15 am](https://user-images.githubusercontent.com/130031825/235560976-0613ae6f-3994-4ff5-98ba-5f5c4029794f.jpg)

**Comment by bofeiw** [comment, marked as answer]
The original rules are:
```
rules_version = '2';
service firebase.storage {
  match /b/{bucket}/o {
    match /{allPaths=**} {
      allow read, write: if request.auth.token.roles.size() > 0;
    }
  }
}
```

**Reply by Ken-JIAnomads** [reply]
I just replaced with above and it worked. Thanks!
